nautobot/dcim/views.py

```diff
@@ -2384,7 +2389,10 @@ def get(self, request, *args, **kwargs):
         if "termination_b_rack" not in initial_data:
             initial_data["termination_b_rack"] = getattr(obj.termination_a.parent, "rack", None)
 
-        form = self.model_form(instance=obj, initial=initial_data)
+        if kwargs.get("termination_b_type", "") == "interface":
+            form = self.model_form(exclude_id=kwargs.get("termination_a_id", ""), instance=obj, initial=initial_data)
+        else:
+            form = self.model_form(instance=obj, initial=initial_data)
```

Comment:
Is it only for interface connections that we should explicitly exclude the termination_a_id? Or should we do so unconditionally for all cases?

Reply:
The form class (`self.model_form`) is set dynamically in the `dispatch` method, so I made it explicit to match this.

Reply:
`self.model_form` is set dynamically in the `dispatch` method so I made this explicit.